sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -22,6 +22,10 @@
 - Removed `Share` from the names of return types and option types.
 - Removed `RequestId` from the return types.
 - Changed `BodyStream` parameter in upload functions from pointer to reference.
+- Renamed `ShareAccessTier` to `AccessTier`.
+- Renamed `ShareRetentionPolicy` to `RetentionPolicy`.
+- Renamed `ShareProtocolSettings` to `ProtocolSettings`.
```

Comment:
Does - Removed Share from the names of return types and option types.` above in the CL covers these changes, or are these different because they are not returned directly (and are not options)? If so, keep these new entries.

Reply:
Those types are not returned directly. They are embedded in the return types.